stwcs/distortion/mutil.py

```diff
@@ -202,7 +202,7 @@ def readIDCtab(tabname, chip=1, date=None, direction='forward',
     refpix['YREF'] = ftab[1].data.field('YREF')[row]
     refpix['XSIZE'] = ftab[1].data.field('XSIZE')[row]
     refpix['YSIZE'] = ftab[1].data.field('YSIZE')[row]
-    refpix['PSCALE'] = round(ftab[1].data.field('SCALE')[row], 8)
+    refpix['PSCALE'] = np.float64(round(ftab[1].data.field('SCALE')[row], 8))
```

Comment:
Can we do this instead?
```suggestion
    refpix['PSCALE'] = round(float(ftab[1].data.field('SCALE')[row]), 8)
```
I was wondering why not convert everything to Python float since these values, seems to me, are used only to update some header values. Would this have some unintended effect?

Reply:
I can make an issue to come back after we've transition to numpy 2.0 and test if changing these has any effect.